python_xbcf/xbcausalforest/xbcf_python.py

```diff
@@ -120,6 +120,7 @@ def __init__(
         sample_weights_flag: bool = True,
         a_scaling: bool = True,
         b_scaling: bool = True,
+        standardize_target = True,
```

Comment:
question: if y standardization is always a good idea, this parameter can be removed and all standardization take place automatically without the option to disable